controls.py

```diff
@@ -0,0 +1,110 @@
+class control():
+    def __init__(self, scale, adjustment, running, list_movements, controller, clock, serial_port, serial_baudrate):
+        self.scale = scale 
+        self.adjustment = adjustment
+        self.running = False
+        self.list_movements = {
+            abutton: 0,
+            bbutton: 0,
+            xbutton: 0,
+            ybutton: 0,
+            left_motor_right: 0,
+            up_motor : 0,
+            right_motor_right: 0,
+            turning_amount: 0
+        }
+        
+        #Initializes the joystick being used, which is the XBox Controller
+        
+        self.controller = pygame.joystick.Joystick(0)
+        #Sets up the timing for the interface
+        
+        self.clock = pygame.time.Clock()
+        
+        self.serial_port = "dev/ttyUSB0"
+        
+        self.serial_baudrate = 9600
+        
+    ser = serial.Serial(port=serial_port, baudrate=serial_baudrate)    
+    #Function that scales the movement on the joystick of the XBox Controller to the amount that needs to be moved by the robot
+    
+    def movement_scaler(self, raw, adjustment):
+        if abs(raw) <= self.adjustment:
+            return 0
+        else:
+            if raw > 0:
+                return (raw-self.adjustment)/1.0
+            elif raw < 0:
+                return  (raw+self.adjustment)/1.0       
+                
+    #Creates variables that store the motor labels for the different motors used. Some motor values have also been put in for adjustments that may be made to add the claw          
```

Comment:
If the below variables are constants, consider using `SCREAMING_SNAKE_CASE` (style recommendation). Also consider putting these variables closer to the top of the module and prepending their names with `_`.